app/code/Magento/Downloadable/Setup/UpgradeData.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
@gelanivishal Thank you for up-porting the issue but for 2.3 this solution should be modified since there we introduced [Declarative Schema](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/) and you shouldn't use `UpgradeData` scripts together with `setup_version` of modules. Please reimplement that

Reply:
@slavvka I have performed changes as requested.